data/.schema.yaml

```diff
@@ -129,13 +129,37 @@ properties:
               enum:
                 - female
                 - male
+                - unknown
             age:
               description: Age of the participant in years.
               oneOf:
                 - type: number
                   minimum: 0
                   maximum: 120
                 - const: unknown
+            race:
+              description: Race of the participant.
+              type: string
+              enum:
+                - asian
+                - black
+                - white
+                - other
+                - unknown
+            ethnicity:
+              description: Ethnicity of the participant.
+              type: string
+              enum:
+                - hispanic
+                - not hispanic
+                - unknown
+            inpatient:
+              description: Inpatient or outpatient status of the participant.
+              type: string
```

Comment:
Thoughts on using `{false, true, "unknown"}` or do we expect another possible status?

Reply:
I currently set it as `enum` with "inpatient", "outpatient", "unknown". In Liu 2024, there is a big difference between inpatient and outpatient sampling protocols. This may not be common information across studies, so I am not sure how to define this.